perun-openapi/openapi.yml

```diff
@@ -14228,7 +14228,7 @@ paths:
       operationId: getBansForResource
       summary: Get all bans for members on the resource.
       parameters:
-        - $ref: '#/components/parameters/resourceId'
+        - { name: resourceId, required: true, description: "Resource id", schema: { type: integer },  in: query }
```

Comment:
How about changing the param name in `ResourcesManagerMethod` instead of here? Then we can still use the component resourceId instead of just duplicating it with a different name.

Reply:
The reason for this was to avoid changing the param name in all other places (mostly CLI) that already send it as "resourceId".

Reply:
![Screenshot from 2022-11-15 10-45-46](https://user-images.githubusercontent.com/97669007/201886801-512e043e-6332-477b-bb1f-37c3ed92bed9.png)
This doesn't look like much. Would you mind fixing this so we are consistent across the board, now that we're going to be working with bans more often?